dockerfiles/compile-extension/build-dd-trace-php

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+set -ex
+
+apk add --no-cache \
+    autoconf \
+    bash \
+    g++ \
+    gcc \
+    libexecinfo-dev \
+    make \
+
+make clean dist_clean
+make all CFLAGS="-std=gnu11 -O2 -Wall -Wextra -Werror" ECHO_ARG="-e"
```

Comment:
We should build with debug symbols. If desired we can always strip them later using `strip --strip-debug`.
```suggestion
make all CFLAGS="-std=gnu11 -O2 -g -Wall -Wextra -Werror" ECHO_ARG="-e"
```